src/components/dashboard/Dashboard.js

```diff
@@ -161,27 +164,135 @@ const Dashboard = props => {
   useRefundDialog(screenProps)
   useInviteCode() // preload user invite code
 
-  const headerAnimateStyles = {
-    position: 'relative',
-    height: headerHeightAnimValue,
+  const profileMinimizedXYAnimValue = new Animated.Value(0)
+  const multiBalanceMinimzedYAnimValue = new Animated.Value(0)
+  const balanceXRange = headerLarge ? [0, -150] : [-150, 0]
+  const altOutputRange = headerLarge ? [0, -150] : [150, 0]
+
+  const balanceMinimizedStyles = {
```

Comment:
move every styles non-related to animations onto getStylesFromProps

Reply:
done, anything not related headerLarge or animations are moved down